pyproject.toml

```diff
@@ -1,3 +1,10 @@
+[tool.ruff]
+line-length = 120
+
+[tool.ruff.per-file-ignores]
+"__init__.py" = ["F401", "E501"]
+"aws_lambda_builders/workflow.py" = ["E501"]
```

Comment:
why ignore E501 on these files?

Reply:
I believe we have higher line lengths allowed by default. This is for an older rule on 79 chars per line.

Reply:
🤔 I thought specifying the line length above would make this rule accommodate to that. If this rule still listens to 79 chars, why would we only ignore it for these files instead of everything?

Reply:
The workflow.py had two lines that I couldn’t find a good way to resolve. So elected to just ignore it. When black runs, everything is fixed anyways.

For ‘__init__.py’, they just do imports so it wasn’t required but was how some other repos did it. Black statement still holds true. 

Our line lengths are 120 as well.